tackle-test-generator-unit/src/org/konveyor/tackle/testgen/core/extender/TestSequenceExtender.java

```diff
@@ -1217,6 +1217,18 @@ else if (SequenceUtil.isMapType(randoopType)) {
                 logger.warning(errmsg);
                 this.extSummary.uncovTestPlanRows__excp__NoSuchMethod++;
                 throw new RuntimeException(errmsg, nsme);
+            } catch (NonInstantiableTypeException nite){ // todo - Exception should be revisited
+                String errmsg = "type is not instrumented for type: " + paramType + "in signature "
```

Comment:
please fix: `type is not instrumented for type` to `type +paramType+ is not instantiable in signature`

Reply:
done